packages/dd-trace/src/appsec/iast/taint-tracking/rewriter.js

```diff
@@ -67,6 +99,10 @@ function disableRewriter () {
   Error.prepareStackTrace = originalPrepareStackTrace
 }
 
+function getOriginalPathAndLineFromSourceMap ({ path, line, column }) {
+  return getRewriterOriginalPathAndLineFromSourceMap(path, line, column)
```

Comment:
This is initialized in `getRewriter()`, so we need to check if `getRewriterOriginalPathAndLineFromSourceMap` is initialized to prevent calls to `undefined(path, line, column)`